resources/schema/Operator.md

```diff
@@ -391,6 +391,40 @@ Length comparison
 
 Complement of `has_equal_length`
 
+### split_parts_have_equal_length
+
+Splits a string by a separator and checks if both parts have equal length. Generic operator for validating paired data formats where both parts must have the same level of detail or precision.
+
+Parameters:
+- `separator`: The delimiter to split on (default: "/")
```

Comment:
You'll need to update rule editor check param for separator as well: https://github.com/cdisc-org/conformance-rules-editor/blob/main/docs/check_parameter.md